applications/StructuralMechanicsApplication/custom_elements/base_solid_element.h

```diff
@@ -954,14 +954,14 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) BaseSolidElement
      * global to local coordinates
      * @param rValues The constitutive laws parameters
      */
-    void RotateToLocalAxes(ConstitutiveLaw::Parameters &rValues);
+    void RotateToLocalAxes(ConstitutiveLaw::Parameters &rValues, KinematicVariables& rThisKinematicVariables);
```

Comment:
```suggestion
    void RotateToLocalAxes(
        ConstitutiveLaw::Parameters &rValues,
        KinematicVariables& rThisKinematicVariables);
```
Minor style guide suggestion.